### PR TITLE
[tabular-data-analysis] add code to git repo

### PR DESCRIPTION
**Pull request by vincentsong-sj**
Add the demo code for tabular data analysis.

### CODE REVIEW

File: tabular-data-analysis/docker-compose-lit.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+services:
+  localai:
+    image: localai/localai:v2.6.1-ffmpeg-core
+    command: --models-path /models --context-size 4096 --threads 2
+    volumes:
+      - $PWD/models:/models
+    ports:
+      - "10080:8080"
+  tabular-data-analysis:
+    image: sunysb/tabular-data-analysis:lit-1.0
```
Comment (feng-ltc) [inline tabular-data-analysis/docker-compose-lit.yaml]: The user should be able to build this image by themselves and use their own image, right? We should only depend on publicly available and well-established images.

Comment (vincentsong-sj) [reply tabular-data-analysis/docker-compose-lit.yaml]: Add run.sh script to build docker image on the fly.

**Comment by feng-ltc** [comment]
lgtm

**Review by feng-ltc — APPROVED** [review]
lgtm

**Pull request merged**